clang/lib/Lex/Preprocessor.cpp

```diff
@@ -35,6 +35,7 @@
 #include "clang/Basic/SourceLocation.h"
 #include "clang/Basic/SourceManager.h"
 #include "clang/Basic/TargetInfo.h"
+#include "clang/Basic/TokenKinds.h"
```

Comment:
I don't think we need that change

Reply:
Agree. Hmm, it's may changed by clangd's auto header insertion functionality.